srcpkgs/kotlin/template

```diff
@@ -0,0 +1,42 @@
+# Template file for 'kotlin'
+pkgname=kotlin
+version=1.2.51
+revision=1
+wrksrc="${pkgname}c"
+only_for_archs="i386 x86_64"
```

Comment:
i686 ?

Reply:
You are right. It's only x86_64. Thanks.